pkg/common/ntpd-rs.service

```
@@ -0,0 +1,18 @@
+[Unit]
+Description=Network Time Service (ntpd-rs)
+Documentation=https://github.com/pendulum-project/ntpd-rs
+After=network-online.target
+Wants=network-online.target
+Conflicts=systemd-timesyncd.service ntp.service chrony.service
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/ntp-daemon
```

Missing disabling of restart
```suggestion
Restart=no
ExecStart=/usr/bin/ntp-daemon
```